src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -169,6 +173,7 @@ C_FUNC(\Name\()_End):
 
         free_stack __PWTB_StackAlloc
         POP_CALLEE_SAVED_REGISTERS
+	// TODO: From this point, r7 is not consistent as FP.
```

Comment:
The instability of atomicity guaranting patch, which is delayed for later PR seems to be caused by using SP-12 at this point. (probably thr caller of this macro reads this instance of stack.)

We can probably avoid such problems by writing r7, sp, pc at SP + 104 and ldm there.